assets/css/_addon/checkbox.scss

```diff
@@ -0,0 +1,14 @@
+.box,
+.box_done,
+.task-list-item {
+  list-style-type: none;
+}
+
+.task-list-item input {
+  margin: 0 0.2em 0.25em -1.6em;
+  vertical-align: middle;
+}
+
+.task-list-item + .task-list-item {
+  margin-top: 3px;
+}
```

Comment:
For now, the new file `checkbox.scss` is unused and should be remove from the PR

Reply:
Whoops, indeed!